tools/cli/defs.go

```diff
@@ -79,13 +79,19 @@ var envKeysForUserName = []string{
 	"HOME",
 }
 
-var resetTypesMap = map[string]string{
+var resetTypesMap = map[string]interface{}{
 	"FirstWorkflowTask":  "",
 	"LastWorkflowTask":   "",
 	"LastContinuedAsNew": "",
 	"BadBinary":          FlagResetBadBinaryChecksum,
 }
 
+var resetReapplyTypesMap = map[string]interface{}{
```

Comment:
This map already exist in generated code. And there are helper to use it in case insensitive fashion.

Reply:
can you use the generated map by proto?

Reply:
generated map does not handle default case, which is reapply signals
```go
var ResetReapplyType_value = map[string]int32{
	"Unspecified": 0,
	"Signal":      1,
	"None":        2,
}
```